src/main/routes/routes.ts

```diff
@@ -678,6 +679,20 @@ export default function (app: Application): void {
         app.get('/cft-rejected-login', app.locals.container.cradle.cftRejectedLoginController.get);
     }
 
+    // SSO Routes
+    if (process.env.ENABLE_SSO === 'true') {
+        app.get('/sso-login', regenerateSession, keepSessionLanguage, (req, res, next) =>
+            passport.authenticate('sso', { failureRedirect: '/' })(req, res, next)
+        );
+
+        app.post(
+            '/sso',
+            (req, res, next) => passport.authenticate('sso', { failureRedirect: '/' })(req, res, next),
```

Comment:
I think regenerateSession needs to be here too

Reply:
This is the redirect URL from the SSO app. regenerateSession is already in the login route 'sso-login'. I don't think it is needed here again.

Reply:
You're right, on checking again I don't think it needs to be here.